webapp/src/main/webapp/themes/cu-boulder/templates/people.ftl

```diff
@@ -182,16 +344,126 @@
             border: 2px solid #c6ebc6;
             padding: 0 10px 10px;
         }
+
+        .caption {
+           margin-left: -10px;
+           margin-right: -11px;
+        }
+.ul {
+    margin: 0 0 4px 19px;
+}
+.navbar {
+  overflow: hidden;
+  background-color: #333;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.navbar a {
+  float: left;
+  font-size: 16px;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+.dropdown {
+  float: left;
+  position: absolute;
+  overflow: hidden;
+}
+
+.dropdown .dropbtn {
+  font-size: 14px;  
+  outline: none;
+  color: black;
+  padding: 4px 6px;
+  background-color: inherit;
+  font-family: inherit;
+  margin: 0;
+  border-radious: 4px;
+  
+}
+
+.navbar a:hover, .dropdown:hover .dropbtn {
+  background-color: grey;
+}
+
+.dropdown-content {
+  display: none;
+  background-color: azure;
+  min-width: 160px;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  z-index: 1;
+  border: 1px solid;
+  width: 600px;
+  padding: 12px;
+}
+
+.dropdown-content a {
+  float: none;
+  color: black;
+  text-decoration: none;
+  display: block;
+  text-align: left;
+  background-color: azure;
+  border: 1px solid;
+  margin: 2px;
+}
+
+
+.dropdown-content a:hover {
+  background-color: #ddd;
+}
+
+.dropdown:hover .dropdown-content {
+  display: block;
+}
+
+ /* Style all font awesome icons */
+.fa-twitter {
+  background: #55ACEE;
+  color: white;
+}
+
+.fa-linkedin {
+  background: #007bb5;
+  color: white;
+}
+.ai-orcid {
+    background: white;
+    color: #A6CE39;
+}
+
     </style>
 
 </head>
 <body>
+  <div class="dropdown">
+    <button 
+      class=btn "dropbtn">help
+      <i class="fa fa-caret-down"></i>
+    </button>
+    <div class="dropdown-content">
+<h3> PEOPLE SEARCH </h3> Use the People Search bar directly below or the expandable Filters at left to explore Boulder faculty data. To limit search to exact terms use double quotes (" ").  Search also allows for wildcard searching by using the wildcard character (*). The filters on the left, such as Research Area, are  only searching the Research Area keywords. To search CU Experts without the filters in order to include all research fields, use the site search bar in the CU Experts page header. Filters default to 'and' logic. Toggle with the 'or' button if desired.
+    </div>
+  </div>
 <div class="facet-view-simple">
-<div class="help"> <h3> PEOPLE SEARCH </h3> Use the People Search bar directly below or the expandable Filters at left to explore Boulder faculty data. People Search allows for wildcard * or exact search with " " double quotations. The research filter is only searching the Research Keywords, not the free-text keywords or research overview. To search VIVO without the filters in order to include all research fields, use the site search bar in the VIVO page header. Filters default to 'and' logic. Toggle with the 'or' button if desired.</div>
+</div>
 </body>
 </html>
+
```

Comment:
Confirm if this extra code is correct... looks like extra /body and /html codes.

Reply:
cleaned up the extra /body and /html along with the orphan </div>
Verified the code looks clean in chrome developer